tests/__init__.py

```diff
@@ -114,3 +123,108 @@ def setUp(self, **environ):
         self.environ.update(environ)
         self.session = create_session()
         self.session.config_filename = 'no-exist-foo'
+
+
+@requires_memory_collection
+class BaseClientDriverTest(unittest.TestCase):
+    def setUp(self):
+        self.runner = ClientRunner()
+        self.runner.start()
+
+    def cmd(self, *args):
+        self.runner.cmd(*args)
+
+    def record_memory(self):
+        self.runner.record_memory()
+
+    def memory_samples(self):
+        return self.runner.memory_samples()
+
+    def tearDown(self):
+        self.runner.stop()
+
+
+class ClientRunner(object):
```

Comment:
The naming convention is a little confusing to me based on the description in ``cmd-runner``. The fact that the code for the "driver" here uses a ``ClientRunner`` while the other file that represents the runner uses the name ``CommandRunner`` makes it hard to differentiate between the classes purposes based on looking at the class name.

Can we name it something like ``CommandDriver`` because it serves as the driver and all of its methods are something like ``send_cmd()`` or ``cmd()``?